main.tf

```diff
@@ -23,16 +23,30 @@ resource "ibm_tg_gateway" "tg_gw_instance" {
 resource "ibm_tg_connection" "vpc_connections" {
   count = length(var.vpc_connections)
 
-  gateway      = local.transit_gateway_id
-  network_type = "vpc"
-  name         = "vpc_conn_inst${count.index}"
-  network_id   = var.vpc_connections[count.index]
+  gateway               = local.transit_gateway_id
+  network_type          = "vpc"
+  name                  = "vpc_conn_inst${count.index}"
+  network_id            = var.vpc_connections[count.index]
+  default_prefix_filter = var.default_prefix_filter
 }
 
 resource "ibm_tg_connection" "classic_connections" {
   count = var.classic_connections_count
 
-  gateway      = local.transit_gateway_id
-  network_type = "classic"
-  name         = "classic_conn_inst${count.index}"
+  gateway               = local.transit_gateway_id
+  network_type          = "classic"
+  name                  = "classic_conn_inst${count.index}"
+  default_prefix_filter = var.default_prefix_filter
```

Comment:
I don't find `default_prefix_filter` in the attributes of https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/tg_connection

Reply:
It is available https://github.com/IBM-Cloud/terraform-provider-ibm/blob/d759bbc59501d0aab305f69fbb0a37865f4d2de5/ibm/service/transitgateway/resource_ibm_tg_gateway_connection.go#L43
but not present in example in TF provider docs.